sandboxes/columnar.txt

```diff
@@ -1,4 +1,4 @@
-# version 7
+# version 8
```

Comment:
similar comment as in `docs/requirements.txt`: The version has changed, but the list of python modules is the same. Why?